src/CoreWCF.Templates/src/templates/CoreWCFService/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿var builder = WebApplication.CreateBuilder();
```

Comment:
Are the \#if's resolved at template creation time or is this the code expected to be seen by the user?

If the goal is to enable all the different scenarios, then I think we should reconsider. There is a balance needed in templates between:
* The least number of lines
* The simplest form of the code
* Handling every possibility

Generally we favor the simplest form of the code - so as to not to scare users off, and to provide a good starting point.

I think we should pick a default for the template, and have that for both the modern form and classic program.cs formats. I would suggest:

| Feature | Included in template |
| --- | --- |
| WSDL | Yes |
| NetTCP | No |
| BasicHttpBinding | Yes |
| Https BasicHttpBinding | Yes |

Reply:
> Are the #if's resolved at template creation time or is this the code expected to be seen by the user?

Yes please see https://github.com/dotnet/templating/wiki/Conditional-processing-and-comment-syntax
> 
> If the goal is to enable all the different scenarios, then I think we should reconsider. There is a balance needed in templates between:
> 
> * The least number of lines
> * The simplest form of the code
> * Handling every possibility
> 
> Generally we favor the simplest form of the code - so as to not to scare users off, and to provide a good starting point.
> 
> I think we should pick a default for the template, and have that for both the modern form and classic program.cs formats. I would suggest:
> 
> Feature	Included in template
> WSDL	Yes
> NetTCP	No
> BasicHttpBinding	Yes
> Https BasicHttpBinding	Yes

Then i will drop netTcp binding support